### DISCUSSION TITLE
Failed to connect to bus after systemctl --user start

### DISCUSSION THREAD

**Discussion by paragasd** [abraunegg/onedrive#1810] [Q&A] [answered]
I seem to get this error: 
```
Failed to connect to bus: Operation not permitted (consider using --machine=<user>@.host --user to connect to bus of other user)
``` 
after using 
```
systemctl --user enable onedrive
``` 

For context, I was also able to login and create a `~/OneDrive` folder, but there isn't a OneDrive folder anywhere in `/home/user/`.  Using `cd ~/OneDrive` changes my directory to `~/OneDrive`, but trying to open any folder in `~/OneDrive` leads to this error message:
```
QStandardPaths: runtime directory '/var/run/user/1000' is not owned by UID 0, but a directory permissions 0700 owned by UID 1000 GID 1000
```
My guess is these error messages are all caused by the same issue, but I'm not sure where to go from here. Any help would be much appreciated.

**Comment by abraunegg** [comment, marked as answer]
@bertaunth
There appears to be a number of issues going on here.

> For context, I was also able to login and create a `~/OneDrive` folder, but there isn't a OneDrive folder anywhere in `/home/user/.`

How did you create this folder? The application, when run as your normal user will create the folder as specified by the 'sync_dir' option in the configuration file - which by default is `~/OneDrive` which, when expanded goes to the logged on users home directory.

Now .. if you are running *any* of these commands via `sudo` - for example: `sudo onedrive` or `sudo mkdir ~/OneDrive` - you are effectively running the application as 'root' not your normal user .. so .. dont use sudo.

> `QStandardPaths: runtime directory '/var/run/user/1000' is not owned by UID 0, but a directory permissions 0700 owned by UID 1000 GID 1000`

This just reinforces my above message. UID 0 = root.

My suggestion here is remove all the directories / configuration you have done (and yes, use sudo to use root privileges to remove / delete) - then start from scratch again and for each of the commands you type - do not use sudo.
